### PR TITLE
Detox Version and Docs Update

### PR DESCRIPTION
**Pull request by MuckT**
### Description

Was running into issues running iOS Detox locally added notes on how to fix this within the `packages/mobile/e2e/README.md`

### Other changes

Detox update to [19.5.7](https://github.com/wix/Detox/releases/tag/19.5.7) - RN 0.67 support

### Tested

Tested locally on iOS and in CI. 

### How others should test

This does not need to be tested by QA. 
Reviewers please check that you are able to run iOS detox tests locally without issue.

### Related issues

N/A

### Backwards compatibility

Yes

### CODE REVIEW

**Review by jeanregisser — COMMENTED** [review]
Thanks! 🙏

File: packages/mobile/package.json [hunk 1]
```diff
@@ -274,12 +274,12 @@
...
         "binaryPath": "ios/build/Build/Products/Debug-iphonesimulator/celo.app",
-        "build": "set -o pipefail && xcodebuild -workspace ios/celo.xcworkspace -scheme celo-test -configuration Debug -sdk iphonesimulator -derivedDataPath ios/build | bundle exec xcpretty",
+        "build": "unset PREFIX && set -o pipefail && xcodebuild -workspace ios/celo.xcworkspace -scheme celo-test -configuration Debug -sdk iphonesimulator -derivedDataPath ios/build | bundle exec xcpretty",
         "type": "ios.app"
       },
       "ios.release": {
         "binaryPath": "ios/build/Build/Products/Release-iphonesimulator/celo.app",
-        "build": "set -o pipefail && xcodebuild -workspace ios/celo.xcworkspace -scheme celo-test -configuration Release -sdk iphonesimulator -derivedDataPath ios/build | bundle exec xcpretty",
+        "build": "unset PREFIX && set -o pipefail && xcodebuild -workspace ios/celo.xcworkspace -scheme celo-test -configuration Release -sdk iphonesimulator -derivedDataPath ios/build | bundle exec xcpretty",
         "type": "ios.app"
```
[the remark this answers is not in the document]
Comment (MuckT) [reply packages/mobile/package.json:283]: It does work in CI without `unset PREFIX`. I can remove it from here and add a note in running the e2e `README.md`

[the remark this answers is not in the document]
Comment (MuckT) [reply packages/mobile/package.json]: Removed this bit from the build script and updated the `packages/mobile/e2e/README.md`.

File: packages/mobile/e2e/README.md [hunk 2]
```diff
@@ -176,6 +176,7 @@ If tests are failing for unknown reasons:
 - Rebuild, re-yarn and rerun. Sometimes the problem just goes away.
 - Delete snapshots in the emulator
 - Look at the emulator while the tests are running. Can you see anything obvious going wrong?
+- If running into issues building for iOS related to nvm try prefixing the build commands in `packages/mobile/package.json` with `unset PREFIX && ...`
```
Comment (kathaypacific) [inline packages/mobile/e2e/README.md:179]: would be cool to mention why this might help here?

**Review by kathaypacific — APPROVED** [review]
🚀 thanks for keeping the detox version always up to date!!

File: packages/mobile/e2e/README.md (hunk 2 above)
Comment (MuckT) [reply packages/mobile/e2e/README.md:179]: Will create a new PR with some doc updates for running in the app in release mode and a more descriptive tidbit about when this change might be needed locally.

**Comment by ValoraQA** [comment]
Hey @muckt JFYI: We are skipping this task as in "How others should test" section it is mentioned N/A.

Thanks !!

**Pull request merged**